### PR TITLE
Automated browser test

### LINKED ISSUE #3: Add browser tests
It would be good to add actual browser tests. We could use the main pipeline that is used in the browser example and test it on a browser matrix, for example with saucelabs.

### PR DESCRIPTION
**Pull request by steabert**
Add a browser test using Cypress. An RTSP server is setup by starting a [gst-rtsp-launch] docker container (https://github.com/steabert/gst-rtsp-launch), combined with a TCP-WebSocket proxy service `utils/tcp-ws-proxy.js`. The `examples/rtsp-ws-browser` example is served with `http-server` to the Cypress test.

This fixes #3

### CODE REVIEW

**Comment by RHammar** [comment]
Perhaps you could add cypress to package.json scripts:
"cypress": "cypress",

So that you can use npm as well as yarn:
npm run cypress run

**Comment by steabert** [comment]
Certainly, I think it could be good to add an actual script that does the whole testing procedure in case one wants to run it locally. I'll look into adding this to the PR.

File: .travis.yml [hunk 1]
```diff
@@ -2,10 +2,18 @@ language: node_js
 node_js:
   - "node"
   - "lts/*"
+services:
+  - docker
 cache: yarn
+before_install:
+  - docker run -d --rm -p 8554:8554 steabert/gst-rtsp-launch
```
Comment (RHammar) [inline .travis.yml]: This container is left alive after the test is complete.
Is it possible to stop the container in after_script?

Comment (steabert) [reply .travis.yml]: I used the guidelines in the [docs](https://docs.travis-ci.com/user/docker/) and there they do not explicitely shut down the container. Is there any recommended way of cleaning up docker containers on Travis?

**Pull request merged**